Odata-docs/odatacli/getting-started.md

```diff
@@ -20,12 +20,12 @@ There are 2 ways to download and install the **OData CLI**
     To install the latest release of the OData CLI [Nuget Package](https://www.nuget.org/packages/Microsoft.OData.Cli/), use the [dotnet tool install](/dotnet/core/tools/dotnet-tool-install) command
 
     ```.NET CLI
-    dotnet tool install --global Microsoft.OData.Cli --version 0.1.0
+    dotnet tool install --global Microsoft.OData.Cli --version 0.3.1
```

Comment:
I don't believe the `--version 0.3.1` is required. I think we should just get rid of it such that we don't have to keep updating the doc every time we release a new version